contents/blog/the-posthog-array-1-11-0.md

```diff
@@ -0,0 +1,76 @@
+---
+date: 2020-07-14
+title: Array 1.11.0
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+
+
+Annotations, filtering all functionality by cohorts, retention table filtering and many, many bugfixes. It's all in this PostHog update.
+
+If you're self hosting and want these features - [update your PostHog](/docs/deployment/upgrading-posthog).
+
+Please also update your `posthog-js` version to at least `1.2.4` if you have it installed via NPM. If you're using the snippet, you're good to go!  
```

Comment:
Let's recommend the latest version: `1.3.1` here (came out Monday, should also be used in the next release).

Changes between 1.2.4 and 1.3.1:

```
## 1.3.1 - 2020-07-13
- Support loading the toolbar with a `__posthog` has param (was: `state`) and `ph_authorize` action.

## 1.3.0 - 2020-07-03
- Add TypeScript defintions
```

Reply:
Actually we should even recommend upgrading to `1.3.2` (just released).

If I recall correctly, master was running on `1.3.1` on Monday. I didn't find a release branch now to double check, however the next release should go out with that version, as it fixes some [promised fixes](https://github.com/PostHog/posthog/issues/1129). 

No need to update the release to 1.3.2, as the fix there only applies to people using it via npm... and they will get the latest version themselves.

Might be good to add another line how it's recommended to update as there are some critical bugfixes.